intermediate_source/tensorboard_profiler_tutorial.rst

```diff
@@ -0,0 +1,10 @@
+PyTorch Profiler With TensorBoard
+====================================
+
+This tutorial has been deprecated.
+
+Redirecting in 3 seconds...
```

Comment:
Instead of redirecting this one, we should add a "use the perfetto with the trace.json instead" comment, similar to the note in https://pytorch.org/tutorials/advanced/cpp_extension.html
<img width="912" alt="image" src="https://github.com/user-attachments/assets/347b28bd-f89d-4d5a-8bf6-376091c5fd52">

The warning I'd imagine would say:

Tensorboard integration with the profiler is now deprecated in favor of using perfetto or the chrome trace for viewing trace.jsons. After [generating a trace](https://pytorch.org/tutorials/recipes/recipes/profiler_recipe.html#using-tracing-functionality), please drag the `trace.json` into [perfetto](https://ui.perfetto.dev/) or chrome://tracing to visualize your profile.